tools/tsp-client/src/commands.ts

```diff
@@ -333,6 +333,7 @@ export async function convertCommand(argv: any): Promise<void> {
   const outputDir = argv["output-dir"];
   const swaggerReadme = argv["swagger-readme"];
   const arm = argv["arm"];
+  const fullyCompatible = argv["fullyCompatible"];
```

Comment:
This needs to be using kebab case like all the other flags
```suggestion
  const fullyCompatible = argv["fully-compatible"];
```